### PR TITLE
CIV-14146 CCD PR - Claimant LR screen: Permission granted by the court

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-14146

### Change description ###
Claimant LR screen: Permission granted by the court

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/CaseField/CaseField-SettleDiscontinue-nonprod.json [hunk 1]
```diff
@@ -141,5 +139,38 @@
...
+    "ID": "permissionGrantedComplex",
+    "Label": " ",
+    "FieldType": "PermissionGrantedFields",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "permissionNotGrantedText",
+    "Label": "### Unable to discontinue this claim \n To discontinue from this claim you need to get permission from the court.\n Click cancel to return to the case summary screen and select 'Make\n an application' from the next steps menu and select 'Other'",
```
Comment (ShwetaTandel-hmcts) [inline ccd-definition/CaseField/CaseField-SettleDiscontinue-nonprod.json:171]: "To discontinue from this claim " should be "To discontinue this claim"

File: ccd-definition/ComplexTypes/DiscontinueClaimClaimant-PermissionGranted.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[
```
Comment (kalachandrasekar1) [inline ccd-definition/ComplexTypes/DiscontinueClaimClaimant-PermissionGranted.json:1]: file name should be _nonprod

File: ccd-definition/CaseField/CaseField-SettleDiscontinue-nonprod.json (hunk 1 above)
Comment (sjamihmcts) [reply ccd-definition/CaseField/CaseField-SettleDiscontinue-nonprod.json]: Done

File: ccd-definition/ComplexTypes/DiscontinueClaimClaimant-PermissionGranted.json (hunk 2 above)
Comment (sjamihmcts) [reply ccd-definition/ComplexTypes/DiscontinueClaimClaimant-PermissionGranted.json]: Done

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by kalachandrasekar1 — DISMISSED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**